instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcTelemetryBuilder.java

```diff
@@ -50,6 +50,7 @@ public final class GrpcTelemetryBuilder {
       additionalClientExtractors = new ArrayList<>();
 
   private boolean captureExperimentalSpanAttributes;
+  private List<String> capturedRequestMetadata;
```

Comment:
Let's make sure this has non-null default value:
```suggestion
  private List<String> capturedRequestMetadata = Collections.emptyList();
```

Reply:
Good idea, done!